### DISCUSSION TITLE
When toggling neotree and have 2 buffers open in vertical split, uneven resizing

### DISCUSSION THREAD

**Discussion by gipo355** [nvim-neo-tree/neo-tree.nvim#1484] [Q&A]
Hey!

I'd like to start using neotree but when i have 2 vertical splits open (pretty often) toggling neotree does this:

![image](https://github.com/nvim-neo-tree/neo-tree.nvim/assets/91525512/4d26d62d-8958-4505-bf8b-091060828396)

I can't code while keeping it open.

Is there a solution to this?

Thank you :pray:

**Comment by bew** [comment]
You may want to enable [`'equalalways'`](https://neovim.io/doc/user/options.html#'equalalways') 🤔
